curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64ddce6f6e8066173208235b.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 64ddce6f6e8066173208235b
+title: Step 27
+challengeType: 20
+dashedName: step-27
+---
+
+# --description--
+
+When `forward` is `False`, the disk has to be displaced in the opposite direction. Write an `else` clause for that. Print the move and change the content of the lists accordingly.
```

Comment:
```suggestion
When `forward` is `False`, the disk has to be moved in the opposite direction. Write an `else` clause for that. Print the move and change the content of the lists accordingly.
```
I would suggest changing this to the word "move" to use the same word from the step in which the `forward` variable was introduced.